components/manager-ip-setter/scripts/manager-ip-setter.sh

```diff
@@ -35,11 +35,13 @@ function set_manager_ip() {
   echo "Restarting services.."
   # Restarting all (except postgres) to avoid issues with not correctly reloading SSL certs
   systemctl restart nginx
-  systemctl restart cloudify-rabbitmq
   systemctl restart cloudify-amqpinflux
   systemctl restart cloudify-influxdb
-  systemctl restart cloudify-restservice
   systemctl restart cloudify-mgmtworker
+  systemctl restart cloudify-rabbitmq
+  systemctl restart cloudify-restservice
+  systemctl restart cloudify-riemann
+  systemctl restart cloudify-stage
```

Comment:
The problem is that in community-edition stage does not exist..
So I guess it would be better to remove it or check if it exists before attempting to restart it..